Import/PaymentAttributeUpdater.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AdyenPayment\Import;
+
+use AdyenPayment\AdyenPayment;
+use Doctrine\Common\Cache\Cache;
+use Shopware\Bundle\AttributeBundle\Service\CrudServiceInterface;
+use Shopware\Bundle\AttributeBundle\Service\TypeMapping;
+use Shopware\Components\Model\ModelManager;
+
+class PaymentAttributeUpdater implements PaymentAttributeUpdaterInterface
+{
+    /** @var CrudServiceInterface */
+    private $crudService;
+    /** @var ModelManager */
+    private $entityManager;
+
+    public function __construct(
+        CrudServiceInterface $crudService,
+        ModelManager $entityManager
+    ) {
+        $this->crudService = $crudService;
+        $this->entityManager = $entityManager;
+    }
+
+    public function setReadonlyOnAdyenTypePaymentAttribute(bool $readOnly)
```

Comment:
this is not a setter method but update `updateReadonlyAdyenType`
`PaymentAttribute` should be in the namespace or in the class name (which you have)